05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,4 +1,56 @@
-.lawn {
+div:first-child {
+  width:500px;
+}
+
+.berryimg>img {
```

Comment:
I was aiming for the div! i know i could've just used the class name, but I wanted to familiarize myself with the "child/parent" relationships in html.